src/cmd/top/joblist_pane.c

```diff
@@ -413,6 +416,24 @@ void joblist_pane_set_current (struct joblist_pane *joblist, bool next)
     }
 }
 
+/*
+ *  Workaround for mvwprintw(3) issues with multibyte jobid 'ƒ' character.
+ *
+ *  Empirically, the JOBID column must be formatted as %12s when the 'ƒ'
+ *  character appears in Flux jobids, but %13s when ascii 'f' is used.
+ *  Guess at the current jobid encoding by determining if jobid 0 has a
+ *  length of 2 (ascii) or 3 (utf-8).
```

Comment:
commit message and this comment confused me until I read the code.  Perhaps "if f58 encoding of jobid 0 has a ..."  otherwise LGTM